docs/scripting/scripts.lib/is_not_today.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: scripts.lib().is_not_today()
+---
+
+## A function that checks if a specified date is not today's date.
```

Comment:
```suggestion
A function that checks if a specified date is not today's date.
```

Reply:
The function actually is bugged and only works correctly if the date is in the past. I'm going to note that in the date param. (take a look at the code to see why)

Reply:
utc_day_diff also needs clarification when that one arrives. It currently only works one direction (first param > second param).

Reply:
note for future: it would be good to give examples of how these functions can be useful for building stuff in game. e.g. this one lets you create experiences that require daily upkeep or can only be done once a day.